specification/vmware/resource-manager/readme.java.md

```diff
@@ -0,0 +1,13 @@
+## Java
+
+These settings apply only when `--java` is specified on the command line.
+Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-libraries-for-java clone>`.
+
+``` yaml $(java)
+java:
+  azure-arm: true
+  namespace: com.microsoft.azure.management.avs
+  override-client-name: AvsClient
+  license-header: MICROSOFT_MIT_NO_CODEGEN
+  output-folder: $(azure-libraries-for-java-folder)/azure-mgmt-avs
+```
```

Comment:
After this, the similar [multi-api](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/appplatform/resource-manager/readme.java.md#java-multi-api) and [tag](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/appplatform/resource-manager/readme.java.md#tag-package-2019-05-01-preview-and-java) should be added.

Reply:
Thanks. I made the change in 3e6ced5.